polynote-frontend/polynote/messaging/receiver.ts

```diff
@@ -601,17 +601,17 @@ export class ServerMessageReceiver extends MessageReceiver<ServerState> {
             }
         });
         this.receive(messages.RunningKernels, (s, kernelStatuses) => {
-            const notebooks: UpdateOf<Record<string, boolean>> = {}
+            const notebooks: string[] = []
```

Comment:
I think we talked about this a little bit... I get the distinction between server's open notebooks and client's open notebooks. But the why does this have to be an array? I think that is likely to cause problems.

Reply:
We discussed this yesterday and concluded that what we have now is fine, though we're worried about the array. If we ever need to watch anything inside this field we'll want to change it to a map or something other than an array.